server/proxy-request.js

```diff
@@ -25,10 +24,39 @@ let passThroughHeaders = function(incomingHeaders) {
 
 
 module.exports = function(url, req) {
-  // support GET or HEAD requests
-  let method = req.method === 'HEAD' ? 'HEAD' : 'GET';
+  // support GET or HEAD or POST requests
+  let methods = ['GET', 'HEAD', 'POST'];
+  let method  = null;
+  if (methods.indexOf(req.method) > -1) {
+    method = req.method;
+  } else {
+    method = 'GET';
+  }
   let externalReqHeaders = passThroughHeaders(req.headers);
 
+  if (method !== 'POST') {
+    externalReqHeaders = u.omit(externalReqHeaders, 'user-agent');
+  }
```

Comment:
'user-agent' header is removed at https://github.com/afeld/jsonp/blob/master/server/proxy-request.js#L16 ; where it is stated that this header is for node to generate.

however, node.js' request library does not generate a user-agent (ref: https://github.com/nodejs/node-v0.x-archive/issues/4552). I am not sure if this has been solved yet, or a configuration issue on my part, but from my tests, the requests library did not.

hence, one way to fix things would be to _not_ remove the 'user-agent' header in function ```passThroughHeaders```; but i'll just wait for your comments on this.